Common/Notifications/NotificationManager.cs

```diff
@@ -78,6 +78,11 @@ public bool Email(string address, string subject, string message, string data =
             {
                 return false;
             }
+            
+            if (headers == null) {
+                headers = new Dictionary<string, string>();
+            }
+            headers['Content-Type'] = 'text/plain';
```

Comment:
Not sure we should override the `Content-Type`, what if the user wants a custom/different content type, maybe this could be the default value if `headers` doesn't contain `'Content-Type'`?

Reply:
Fixed.